### DISCUSSION TITLE
Can I put model definition in a file and use it when blitz generate ?

### DISCUSSION THREAD

**Discussion by KiritoXF** [blitz-js/blitz#2338] [Q&A] [answered]
Hi there, I'm new to blitz-js and feel a little confussed when i read the `blitz generate` documentation.
When i want to generate Model, Queries, Mutations or Pages, i have to execute a command like this:
```
 blitz generate all tasks \
       name:string \
       completed:boolean:default=false \
```
It seems that i have to type all the model fields' definition in the command line and execute it.
Is there any place else to put the model definition in to make team-work easier? ( or did i miss something?)
For example, `blitz generate all model_schema_file_name`

It will be greate help if anyone could give some advice.
Thanks!

**Comment by flybayer** [comment, marked as answer]
You can add the model schema directly to `db/schema.prisma` if you just want to add the model without generating code or using the CLI.

**Reply by KiritoXF** [reply]
Thanks! I guess this is what i need.
